### ISSUE TITLE
Alterar o otimizador utilizado na etapa de compilação da rede neural

### ISSUE THREAD

**Issue by HaraHeique** [HaraHeique/TCC-rede-neural-siamesa#15]
Trocar o otimizador ADAM pelo RMSProp.

**Comment by HaraHeique** [comment]
Links de referências:

- https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/RMSprop;

**Comment by HaraHeique** [comment]
Basicamente não mudou quase nada em relação ao otimizadador **ADAM**, o qual era o que já existia anteriormente. Colocarei os gráficos depois novamente utilizando ele como otimizador.

**Comment by HaraHeique** [comment]
## Otimizador RMSProp

### Com constrastive loss
![history-graph-contrastive_loss-RMSProp-70-30-100-14](https://user-images.githubusercontent.com/26990166/101268015-978c0200-373d-11eb-8dd7-019504d659a2.png)

### Com mean_squared_error loss
![history-graph-mean-loss-RMSProp-70-30-100-14](https://user-images.githubusercontent.com/26990166/101268480-269b1900-3742-11eb-852a-d32a5d07b181.png)

**Comment by HaraHeique** [comment]
### 70/30 MSE RMSProp

![history-graph-70-30-100-5-mse-rmsprop](https://user-images.githubusercontent.com/26990166/102692700-0c9e1380-41e3-11eb-96bb-329617722c31.png)
